inc/views/base/modcp/warninglogs.twig

```diff
@@ -0,0 +1,119 @@
+{% extends 'layouts/master.twig' %}
+
+{% block head %}
+    <title>{{ mybb.settings.bbname }} - {{ lang.warning_logs }}</title>
+{% endblock head %}
+
+{% block body %}
+    <table width="100%" border="0" align="center">
+        <tr>
+            {# To do: modcp_nav #}
+            <td valign="top">
+                <table border="0" cellspacing="{{ theme.borderwidth }}" cellpadding="{{ theme.tablespace }}" class="tborder">
+                    <tr>
+                        <td class="thead" align="center" colspan="6"><strong>{{ lang.warning_logs }}</strong></td>
+                    </tr>
+                    <tr>
+                        <td class="tcat" width="15%"><span class="smalltext"><strong>{{ lang.warned_user }}</strong></span></td>
+                        <td class="tcat" width="25%"><span class="smalltext"><strong>{{ lang.warning }}</strong></span></td>
+                        <td class="tcat" width="20%" align="center"><span class="smalltext"><strong>{{ lang.date_issued }}</strong></span></td>
+                        <td class="tcat" width="20%" align="center"><span class="smalltext"><strong>{{ lang.expires }}</strong></span></td>
+                        <td class="tcat" width="15%"><span class="smalltext"><strong>{{ lang.issued_by }}</strong></span></td>
+                        <td class="tcat" width="1%" align="center"><span class="smalltext"><strong>{{ lang.details }}</strong></span></td>
+                    </tr>
+                    {% for warning in warning_list %}
+                        {% include 'modcp/warninglogs_logitem.twig' %}
```

Comment:
Might make sense to instead use `modcp/warninglogs/item.twig` - we can nest templates into folders which IMO is easier to read and search.